### PR TITLE
Bug fixing and updating tests for review cl

### PR DESCRIPTION
**Pull request by mhkhan27**

### CODE REVIEW

**Comment by mhkhan27** [comment]
[compare_cl_with_datasets - review_xx](https://trello.com/c/jYqNxCLV/147-compareclwithdatasets-reviewxx)

File: R/review_cleaning_log.R [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+
+# generate cleaning log ---------------------------------------------------
+create_cleaning_log <- function(raw_data = raw_data,
+                                raw_data_uuid = "X_id",
+                                clean_data = clean_data,
+                                clean_data_uuid = "X_id",
+                                check_for_deletion_log = T,
+                                check_for_variable_name = T) {
+  my_bind_row <- get("bind_rows",asNamespace("dplyr"))
```
[the remark this answers is not in the document]
Comment (mhkhan27) [reply R/review_cleaning_log.R:29]: dplyr::bind_rows didnt work.

**Comment by mhkhan27** [comment]
> 

Added test for all scenario- 

1: no change, << test line 715-769
2: change applied - value to value, << test line 620-661
3: change applied NA to value,<< test line 620-661
4: change applied value to NA, << test line 620-661
5: change not applied value to value, << test line 567-616
6: change non applied Na to value, << test line 567-616
7: changed not applied value to NA. << test line 567-616
8: deleted variable -> not showing << intentional
9: added variable -> not showing << intentional
10: duplicated entries -> ok 
11: deletion: in deleted log, not in clean data << test line 207-253
12: deletion: in deleted log, in clean data << test line 476-535
13: deletion: in raw data, not in clean data, not in deleted log << test line 441-468

**Comment by mhkhan27** [comment]
Fixed the issue. Your test is also added to line no 272 to 328.

**Review by yannsay-impact — APPROVED** [review]

**Pull request merged**